libqtile/group.py

```diff
@@ -256,6 +256,8 @@ def add(self, win, focus=True, force=False):
             win._float_state = FloatStates.FULLSCREEN
         elif self.floating_layout.match(win) and not win.fullscreen:
             win._float_state = FloatStates.FLOATING
+            if not win.kept_above and self.qtile.config.floats_kept_above:
+                win.cmd_keep_above()
```

Comment:
We can change this to `keep_above()` now ;). Might be worth grepping, I saw another use of `cmd_` too.